src/js/classes/AjaxBootstrapSelectRequest.js

```diff
@@ -47,6 +47,10 @@ var AjaxBootstrapSelectRequest = function (plugin) {
             q: '{{{q}}}'
         };
     }
+    
+    if (this.options.ajax && this.options.ajax.url && $.isFunction(this.options.ajax.url)) {
```

Comment:
This is wrong. It should be checking if `this.options.url` exists and is a function.

The `this.options` in this context is for the `AjaxBootstrapSelectRequest` class, not the plugin's options:

See https://github.com/truckingsim/Ajax-Bootstrap-Select/pull/102/files#diff-192c7a5fedc370a66ab88b12003d5f23L30

```js
this.options = $.extend(true, {}, plugin.options.ajax);
```